CHANGELOG.md

```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.5.0] - 2023-04-02
+### Fixed
```

Comment:
Please change the order as follows (Added, Changed, etc.):
Ref: https://keepachangelog.com/en/1.1.0/#how
```
### Added
- Add hyperlink to Privacy Policy in footer. [#997](https://github.com/rokwire/events-manager/issues/997)
- Update the style of the settings page in the campus event. [#994](https://github.com/rokwire/events-manager/issues/994)
- Update the style for buttons on add new event page. [#996](https://github.com/rokwire/events-manager/issues/996)
- Update the style for white background buttons on hover. [#1006](https://github.com/rokwire/events-manager/issues/1006)
- Add the version to footer. [#988](https://github.com/rokwire/events-manager/issues/988)
- Add the syle to the dropdown menu in user event create page. [#1039](https://github.com/rokwire/events-manager/issues/1039)

### Changed
- Change the GUI of text input field box. [#991](https://github.com/rokwire/events-manager/issues/991)
- Update the style of pagination in user event listing page. [#989](https://github.com/rokwire/events-manager/issues/989)
- Update the home page icon and events manager hyperlink redirection. [#1008](https://github.com/rokwire/events-manager/issues/1008)
- Update Login and Logout button. [#1009](https://github.com/rokwire/events-manager/issues/1009)
- Update the buttons in the user event edit page. [#1013](https://github.com/rokwire/events-manager/issues/1013)
- Update the publish button and pending button on the campus event detail page. [#1014](https://github.com/rokwire/events-manager/issues/1014)
- Update the style of the "Link a Sub-Event" button. [#1015](https://github.com/rokwire/events-manager/issues/1015)
- Update the style of all checkboxes. [#1022](https://github.com/rokwire/events-manager/issues/1022)
- Update the hover and focus style of all buttons. Refactored and cleaned unused styles. [#1048](https://github.com/rokwire/events-manager/issues/1048)
- Update the dropdown menu on user list page. [#1063](https://github.com/rokwire/events-manager/issues/1063)
- Update the style of modals/pop-up windows. [#1054](https://github.com/rokwire/events-manager/issues/1054)
- Update the style of reset button. [#1061](https://github.com/rokwire/events-manager/issues/1061)
- Update the placeholder for user events. [#1062](https://github.com/rokwire/events-manager/issues/1062)

### Fixed
- Fix the color of publish, pending and hide the past. [#995](https://github.com/rokwire/events-manager/issues/995)
- Fix the style of badge and checkbox of the filter. [#1011](https://github.com/rokwire/events-manager/issues/1011)
- Fix the height of Create Event button. [#1021](https://github.com/rokwire/events-manager/issues/1021)
- Fix the hover text decoration and color of Edit button in the event detail page. [#1023](https://github.com/rokwire/events-manager/issues/1023)
- Fix the scheduler to take the input from setting page. [#1030](https://github.com/rokwire/events-manager/issues/1023)
- Fix the display of the orange line under the dropdown menu. [#1042](https://github.com/rokwire/events-manager/issues/1042)
- Fix flash message and group building block failure. [#1075](https://github.com/rokwire/events-manager/pull/1075)
- Fix the mouse shape to be pointer when hovering on buttons. [#1059](https://github.com/rokwire/events-manager/issues/1042)
- Fix the round corners for the input text fields. [#1060](https://github.com/rokwire/events-manager/issues/1060)
- Fix the required fields validation when creating a new user event. [#1072](https://github.com/rokwire/events-manager/issues/1072)
- Fix the style of add icon. [#1041](https://github.com/rokwire/events-manager/issues/1041)
- Fix Detect secrets GitHub action build error and updated baseline. [#1070](https://github.com/rokwire/events-manager/issues/1070)
```

Reply:
fixed

Reply:
Thanks.